android/2014-06-04-check-activities.md

```diff
@@ -8,15 +8,15 @@ categories:
 order: 603
 ---
 
-## Details 
+## Details
 
-Typically in Android applications an Activity is a “Screen” in an app. An activity can be invoked by any application if it is exported, thus allowing the user access to regions of the application often in unintended ways. If you define an Intent filter for an Activity it will automatically make it public and accessible by other applications. Even if an Activity is not public it can still be invoked with root privileges. This could allow an attacker to jump around an application in a way the developer did not intend, such as jumping past a password lock screen to access the data.
+An Activity can be invoked by any application if it is [`exported` and `enabled`](http://developer.android.com/guide/topics/manifest/activity-element.html). This could allow an attacker to load UI elements in a way the developer may not intend, such as jumping past a password lock screen to access data or functionality. By default Activities are not exported, however, if you define an Intent filter for an Activity, by default it will automatically be exported by the system.
```

Comment:
there are two "by default" in the last sentence. I'd probably remove the second one.